.github/workflows/builds.yml

```diff
@@ -12,111 +12,97 @@ jobs:
       fail-fast: false
       matrix:
         include:
+        # Debug templates
           - platform: linux
             float-precision: single
             arch: x86_64
+            target-type: template_debug
             os: ubuntu-20.04
+
           - platform: windows
             float-precision: single
             arch: x86_32
+            target-type: template_debug
             os: windows-latest
+
           - platform: windows
             float-precision: single
             arch: x86_64
+            target-type: template_debug
             os: windows-latest
+
           - platform: macos
             float-precision: single
             arch: universal
+            target-type: template_debug
             os: macos-latest
+
           - platform: android
             float-precision: single
             arch: arm64
+            target-type: template_debug
             os: ubuntu-20.04
+
           - platform: android
             float-precision: single
             arch: arm32
+            target-type: template_debug
             os: ubuntu-20.04
+
           - platform: android
             float-precision: single
             arch: x86_64
+            target-type: template_debug
             os: ubuntu-20.04
+
           - platform: android
             float-precision: single
             arch: x86_32
+            target-type: template_debug
             os: ubuntu-20.04
+
           - platform: ios
             float-precision: single
             arch: arm64
+            target-type: template_debug
             os: macos-latest
+
           - platform: web
             float-precision: single
             arch: wasm32
+            target-type: template_debug
             os: ubuntu-20.04
 
+        # Release templates
           - platform: linux
-            float-precision: double
-            arch: x86_64
-            os: ubuntu-20.04
-          - platform: windows
-            float-precision: double
-            arch: x86_32
-            os: windows-latest
-          - platform: windows
-            float-precision: double
+            float-precision: single
             arch: x86_64
-            os: windows-latest
-          - platform: macos
-            float-precision: double
-            arch: universal
-            os: macos-latest
-          - platform: android
-            float-precision: double
-            arch: arm64
+            target-type: template_release
             os: ubuntu-20.04
-          - platform: android
-            float-precision: double
-            arch: arm32
-            os: ubuntu-20.04
-          - platform: android
+
+        # Double precision templates
+          - platform: linux
             float-precision: double
             arch: x86_64
-            os: ubuntu-20.04
-          - platform: android
-            float-precision: double
```

Comment:
I think you deleted the `double` templates and the `template_release` templates for most platforms here.

Reply:
Ah, I wasn't sure if I should include them all since they'd create a ton of builds per push. Should I include all possible combinations?

Reply:
Would be great to have the same combinations as before. After all this is a template and not the GDExtension itself. We could comment out the `double` builds perhaps to reduce the number of builds but in general they should be there in case someone wants to enable them

Reply:
Yeah that sounds good